requirements_devel.txt

```diff
@@ -6,6 +6,4 @@
 -r requirements.txt
 
 # Development packages
-pep8
```

Comment:
You should not have removed these packages from the developer requirements

Reply:
I thought it was much prudent to remove them from the requirements_devel.txt since the same requirements are already repeated in the requirements.txt file.I just thought this violates the DRY principal.